main/templates/stock/list.html

```diff
@@ -0,0 +1,82 @@
+{% extends "master.html" %}
+
+
+{% block contenido %} 
+<!--PANEL-->
+<div class="col-lg-4 bg-warning fixed-bottom d-lg-flex d-none flex-column justify-content-center border-end border-dark" style="top: 2.5vh;">
+    <h1 class="px-5">Gesti&oacute;n de stock</h1>
+    <h5 class="px-5">Registro y seguimiento de los bienes y suministros.</h5>
+</div>
+
+<!-- REST OF THE PAGE -->
+<div class="container-fluid h-100">
+    <!-- PANEL FOR SMALL SCREENS -->
+    <div class="row d-lg-none d-flex bg-warning border-dark border-top border-bottom mb-3">
+        <div class="col-12 d-flex justify-content-center">
+            <h1 class="px-5">Gesti&oacute;n de stock</h1>
```

Comment:
Is this a good practice to not use utf8 symbol and use "&oacute;" instead?

Reply:
Actually, I don't know. I know that sometimes spaces aren't shown unless you use &nbsp; so whenever I remember I try to use that, just in case :) How do you comment on specific code? That's cool, I want to know that 👀

Reply:
Ok, it's not really a relevant thing I think 👍🏻. Referring to the comments, if you are in a review or a pull request, in the "files changed" tab you can click in any line of code and write the comments you want. I hope you find it useful 😄